.config/.eslintrc.cjs

```diff
@@ -0,0 +1,72 @@
+const ts = require("@typescript-eslint/eslint-plugin");
+const js = require("@eslint/js");
+
+const ts_rules_disabled = Object.keys(ts.rules).reduce(
+	(acc, rule) => ({
+		...acc,
+		[`@typescript-eslint/${rule}`]: "off"
+	}),
+	{}
+);
+
+const js_rules_disabled = Object.keys(js.configs.all.rules).reduce(
+	(acc, rule) => ({
+		...acc,
+		[`${rule}`]: "off"
+	}),
+	{}
+);
+
+module.exports = {
+	parser: "@typescript-eslint/parser",
+	parserOptions: {
+		project: "./tsconfig.json",
+		extraFileExtensions: [".svelte"] // This is a required setting in `@typescript-eslint/parser` v4.24.0.
+	},
+	env: {
+		browser: true,
+		node: true
+	},
+	overrides: [
+		{
+			files: ["*.svelte"],
+			parser: "svelte-eslint-parser",
+			// Parse the `<script>` in `.svelte` as TypeScript by adding the following configuration.
+			parserOptions: {
+				parser: "@typescript-eslint/parser"
+			}
+		}
+	],
+	extends: [
+		// add more generic rule sets here, such as:
+		"eslint:recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:svelte/recommended"
+	],
+	rules: {
+		...ts_rules_disabled,
+		...js_rules_disabled,
+		"no-console": [2, { allow: ["warn", "error"] }],
+		"no-constant-condition": 2,
```

Comment:
Maybe "error" instead of the seemingly magic number 2? :D

Reply:
I guess that would make more sense. :D